### DISCUSSION TITLE
Dev/stage environment

### DISCUSSION THREAD

**Discussion by cruiservlad** [chaskiq/chaskiq#738] [Q&A]
Hello! I have the next issue when try to run app in my remote test server:

> Cannot render console from 46.173.179.76! Allowed networks: 127.0.0.0/127.255.255.255, ::1
>   ActiveRecord::SchemaMigration Pluck (2.4ms)  SELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC
> Processing by ArticlesController#show as HTML
> FATAL: Listen error: unable to monitor directories for changes.
> Visit https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers for info on how to fix this.
> Completed 500 Internal Server Error in 1240ms (ActiveRecord: 0.0ms | Allocations: 825927)
> 
> 
> Not notifying due to an invalid api_key
> 
> SystemExit (FATAL: Listen error: unable to monitor directories for changes.
> Visit https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers for info on how to fix this.
> ):
> 
> listen (3.0.8) lib/listen/adapter/linux.rb:34:in `abort'

![image](https://user-images.githubusercontent.com/11313390/168457636-7d0bc51d-7c85-4e18-ad1a-6d274f5588ad.png)

What I do wrong? I try run dev env in remote VPS.

**Comment by michelson** [comment]
Hi, don't know really what could be but this link may work for you.
https://stackoverflow.com/questions/42225677/listen-error-unable-to-monitor-directories-for-changes
TLDR; remove the Gemfile.lock and run `bundle`, then restart the server again.
